platformio.ini

```diff
@@ -193,6 +193,18 @@ lib_deps        = https://github.com/Xinyuan-LilyGO/TTGO-T-Display
 build_flags     = -DUNITY_INCLUDE_DOUBLE
                   -DUNITY_DOUBLE_PRECISION=1e-12    ; Make doubles real 8 byte doubles
 
+[matrix_config]
```

Comment:
Only catch I can see here is that Weather isn't actually dependent on running on a MATRIX.   With enough pixels, it could run on a WS2812B array as well.  But I don't forsee that happening any time soon.

Reply:
That's a good point. If and when that changes we'll have to restructure this and other things. For one, in the NightDriverStrip source code, the inclusion of all PatternXxxx effects is conditional on USE_MATRIX being set.